syntax = "proto3";

package example2;

service Example2 {
    rpc CreateData2(CreateData2Request) returns (CreateData2Response) {}
    rpc ReadData2(ReadData2Request) returns (ReadData2Response) {}
}

message CreateData2Request {
    string header = 1;
    string data = 2;
}

message CreateData2Response {
    string message = 1;
    int32 code = 2;
}

message ReadData2Request {
    string id = 1;
}

message ReadData2Response {
    string header = 1;
    string data = 2;
    string message = 3;
    int32 code = 4;
}